compiler-rt/test/asan/TestCases/wcsncpy.cpp

```diff
@@ -0,0 +1,51 @@
+// RUN: %clang_cl_asan -Od -Zi %s -Fe%t

```

Comment:
This appears to be basically the same as the below `// RUN: %clangxx_asan -O0 %s -o %t && not %run %t 2>&1 | FileCheck %s`, I don't think we need it (debug information is the default)